### DISCUSSION TITLE
API usage from external app

### DISCUSSION THREAD

**Discussion by bkbilly** [wolveix/satisfactory-server#322] [Q&A] [answered]
I want to get some information about my server, like the Number of Connected Players.
I found that there is an API that uses the [QueryServerState](https://github.com/wolveix/satisfactory-server/wiki/Official-API-Docs#queryserverstate) that can return the information I want, but I am not sure how to use it.

Could you help me figure out the command I need to run for example with CURL?

**Comment by wolveix** [comment, marked as answer]
@bkbilly please read https://github.com/wolveix/satisfactory-server/wiki/Official-API-Docs

You can see how we use the API in our `healthcheck.sh` script: https://github.com/wolveix/satisfactory-server/blob/f3425aeed513db2fe264bbec21a8dba0fc52db27/healthcheck.sh

**Comment by bkbilly** [comment]
I read the docs, as I have already mentioned on my post.
The `healthcheck.sh` script helped, but now I get an error.
This is my input:
```json
{
	"function":"QueryServerState"
}
```
This is the response:
```json
{
	"errorCode": "insufficient_scope",
	"errorMessage": "The client is missing required privileges to access the given function"
}
```

**Reply by wolveix** [reply]
@bkbilly it is not my API, please ask for support in the appropriate channels in the Satisfactory Discord server :)
